context.go

```diff
@@ -858,26 +869,32 @@ func (c *Context) findSubdirBlueprints(dir string, subdirs []string, subdirsPos
 	subBlueprintsName string, optional bool) (blueprints, deps []string, errs []error) {
 
 	for _, subdir := range subdirs {
-		globPattern := filepath.Join(dir, subdir)
-		matches, matchedDirs, err := pathtools.Glob(globPattern)
+		pattern := filepath.Join(dir, subdir)
```

Comment:
In this case, our pattern should include subBlueprintsName so that we don't add a dependency on the subdir if it doesn't exist.